tarpc/tests/dataservice.rs

```diff
@@ -0,0 +1,57 @@
+use futures::{future, prelude::*};
+use std::io;
+use tarpc::{
+    client, context,
+    server::{self, Handler},
+};
+
+#[cfg(all(feature = "serde1", feature = "tokio1"))]
```

Comment:
Rather than put the cfgs in the test itself, you can put them in the Cargo.toml. Something like this:

```toml
[[test]]
name = "dataservice"
required-features = ["serde1", "tokio1"]
```

Reply:
Done :)